config.example.yaml

```
@@ -1,21 +1,132 @@
-address: 127.0.0.1:8888
+############ Configuration Example ############
+# This contains an example configuration file for OpenMeter OSS.
+# Values here match the default configuration, unless noted otherwise.
+###############################################
 
-telemetry:
-  log:
-    level: debug
-
-# dedupe:
-#   enabled: true
-#   driver: redis
-#   config:
-#     address: 127.0.0.1:6379
-#     database: 0
-#     expiration: 768h # 32d
-
-# Consumer portal
+
+###### Server configuration ######
+## Address the server listens on
+address: 127.0.0.1:8888 # Default: :8888
+
+# ## Aggregation configuration
+# # default values
+# aggregation:
+#     ## ClickHouse connection configuration
+#     clickhouse:
+#         address: 127.0.0.1:9000
+#         database: openmeter
+#         password: default
+#         tls: false
+#         username: default
+#
+
+## Deduplication of metering events
+dedupe:
+    # ## Redis connection config, only used when driver=redis
+    # config:
+    #     address: 127.0.0.1:6379
+    #     database: 0
+    #     expiration: 24h
+    #     password: ""
+    #     sentinel:
+    #         enabled: false
+    #         mastername: ""
+    #     size: 128
+    #     tls:
+    #         enabled: false
+    #         insecureskipverify: false
```

```suggestion
    #         insecureSkipVerify: false
```

Camel case looks nicer, Viper lowercases everything anyway.